### DISCUSSION TITLE
Make on deploy?

### DISCUSSION THREAD

**Discussion by kaihendry** [serverless/serverless#10473] [Q&A]
Is there a way (like in AWS SAM) to trigger a Makefile on deployment, or do you have use a Makefile instead, like I do here?

https://github.com/kaihendry/sls-nodejs-to-go/blob/main/Makefile

**Comment by pgrzesik** [comment]
Hello @kaihendry - could you show how SAM handles that? It definitely can be done with plugins but I'd like to better understand the problem first

**Reply by kaihendry** [reply]
https://github.com/kaihendry/days/blob/main/template.yml#L48

**Reply by pgrzesik** [reply]
There's no native for such functionality and I'm not aware of any plugins that do that
